SwiftSyft/Classes/SyftWebSocket_iOS12.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  SyftWebSocket.swift
+//  SwiftSyft
+//
+//  Created by Sasha Bataieva on 25/01/2020.
+//
+
+import Foundation
+import Starscream
+
+public class SyftWebSocketIOS12: SyftWebSocket, WebSocketDelegate {
```

Comment:
You don't need to inherit `SyftWebSocket` anymore, just implement `SocketClientProtocol` directly.

Reply:
fixed